src/Host.cpp

```diff
@@ -1508,18 +1508,20 @@ void Host::processGMCPDiscordStatus(const QJsonObject& discordInfo)
     auto largeImages = discordInfo.value(QStringLiteral("largeimage"));
     if (largeImages != QJsonValue::Undefined) {
         auto largeImage = largeImages.toArray().first();
+        // TODO: We do not cache this list of icons - perhaps we should as then we can validate requests against it
```

Comment:
Please don't add TODO's, let's use the wishlist on Github for ideas (and discussion)

Reply:
Raised as Issue #2350 .